### ISSUE TITLE
Elevated urine creatinine concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6030]
**Preferred term label:** Elevated urine creatinine concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of creatinine in the urine is above the normal range., PMID: 27717425

**Parent term (use hpo.jax.org/app)** Aminoaciduria HP:0003355

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Creatinine concentration is always dependent on context -- there is no way of saying whether it is too high or too low since the concentration changes with the concentration of the urine. There is no way of coding this with a single term, this is a limitation of this kind of phenotype ontology.
